CHANGELOG.md

```diff
@@ -31,6 +31,8 @@ Changes
 * [UI/Developer]: Updated to latest (3.26.0) release of `ant.design`.
 * [Developer]: Fixed url for downloading multiple analysis outputs as well as for viewing the user analysis outputs page.
 * [UI/Developer]: Lazy load line list export utilities.
+* [Developer]: Update to Chromedriver 79.0.0 for Travis CI.
+* [UI/Developer]: Standardized JavaScript internationalization through webpack.
```

Comment:
Link to the location in the docs for the info about this change please.

Reply:
Updated in d4f9fd136a0975aed4b5604707527a80b808dfab